vulture_os/system/cluster/models.py

```diff
@@ -379,9 +379,8 @@ def get_listeners_enabled(self):
     @property
     def get_forwarders_enabled(self):
         """ Return all tuples (family, proto, ip, port) for each LogForwarders """
-        from applications.logfwd.models import LogOM, LogOMRELP, LogOMHIREDIS, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
+        from applications.logfwd.models import LogOM, LogOMRELP, LogOMFWD, LogOMElasticSearch, LogOMMongoDB, LogOMKAFKA
```

Comment:
I am not truely ok with this change, we need this import but it is declared also at line 47.
A switch would be better.

Reply:
Corrected : imports are back to the top of the file